src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx

```diff
@@ -174,7 +178,9 @@ const NoData = ({
       </PanelTitle>
     )}
   >
-    No data found
+    <div className="quiet">
+      {`No ${providers[provider].name} data found.`}
```

Comment:
I worry a little bit about repetition with the title above!

Reply:
Actually… there's a very good point that I didn't even look at the titles! I'm not sure I love using it for status text ("Invalid Provider") or for the provider name during loading. It feels a little strange to me, but I want to muse on it for a few more minutes.

Regardless, I don't mind the repetition so much, and like the explicitness of the provider name in the message, unless you hard disagree!

Reply:
I came down on "you're right"!